addons/website_quote/controllers/main.py

```diff
@@ -31,6 +31,10 @@ def view(self, order_id, pdf=None, token=None, message=False, **post):
         if not Order:
             return request.render('website.404')
 
+        # Token or not, sudo the order, since portal user has not access on
+        # taxes, required to compute the total_amout of SO.
+        Order = Order.sudo()
```

Comment:
If sudoing is necessary, variable name should be changed in order to avoid confusion.